public/members.json

```diff
@@ -42,5 +42,9 @@
   {
     "member": "Pragya Rajpurohit",
     "url": "https://pragyaraj.tech/index1.html"
+  },
+  {
+    "member": "Tanishq Soni",
+    "url": "https://tanishqsoni.me"
```

Comment:
This site doesn't seem to have the widget

Reply:
which widget are u talking about?

Reply:
read the instructions here: https://webring.hackclub.com

Reply:
can you check it now, at the bottom of the website !

Reply:
should be good